custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -1504,10 +1508,10 @@ BEGIN
 		'month, ' ||
 		_rollup_text ||
 		'4, ' ||
-		quote_nullable(_null_value) || ', ' ||
-		quote_nullable(_null_value) || ', ' ||
-		quote_nullable(_null_value) || ', ' ||
-		'1, ' ||
```

Comment:
Based on whether we want to keep these values as null or not, it might just cleaner to just replace the "1" line with "CASE WHEN sum(num_launched_X) > 0 ...." for each level of aggregation.

Reply:
Ah nevermind, I see the reason for setting these explicitly to zero.  This looks good to me then.